resources/metadata/abapAddonAssemblyKitCheckCVs.yaml

```diff
@@ -0,0 +1,68 @@
+metadata:
+    name: abapAddonAssemblyKitCheckCVs
+    description: This step checks the validity of Software Component Versions.
+    longDescription: |
+      This steps takes a list of Software Component Versions from the addonDescriptorFileName and checks whether they exist or are a valid successor of an existing Software Component Version.
+      It resolves the dotted version string into version, support package level and patch level and writes it to the commonPipelineEnvironment.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapAddonAssemblyKitCredentialsId
+        description: Credential stored in Jenkins for Addon Assembly Kit as a Service System (AAKaaS)
+        type: jenkins
+    params:
+      - name: abapAddonAssemblyKitEndpoint
+        type: string
+        description: Base URL to the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        - GENERAL
+        mandatory: true
+      - name: username
+        type: string
+        description: User for the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        secret: true
+      - name: password
+        type: string
+        description: Password the Addon Assembly Kit as a Service System (AAKaaS)
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        secret: true
+      - name: addonDescriptorFileName
+        type: string
+        description: File name of the YAML file which describes the Product Version and corresponding Software Component Versions
+        mandatory: true
+        default: addon.yml
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: addonDescriptor
```

Comment:
Just an idea, but can `addonDescriptor` and `addonDescriptorFileName` be combined that they both use the same format and thus we only need one parameter? It sounds as it contains the same information.

Reply:
As we plan an refactoring anyway I would take this as action item for an later PR